tests/src/gui/testprocessinggui.cpp

```diff
@@ -762,6 +764,89 @@ void TestProcessingGui::testStringWrapper()
   delete l;
 }
 
+void TestProcessingGui::testAuthCfgWrapper()
+{
+  QgsProcessingParameterAuthConfig param( QStringLiteral( "authcfg" ), QStringLiteral( "authcfg" ) );
+
+  // standard wrapper
+  QgsProcessingAuthConfigWidgetWrapper wrapper( &param );
+
+  QgsProcessingContext context;
+  QWidget *w = wrapper.createWrappedWidget( context );
+
+  QSignalSpy spy( &wrapper, &QgsProcessingAuthConfigWidgetWrapper::widgetValueHasChanged );
+  wrapper.setWidgetValue( QStringLiteral( "xxx" ), context );
+  QCOMPARE( spy.count(), 1 );
+
+  // hard to test these - we don't have a standard test authcfg to set to
+  // QCOMPARE( wrapper.widgetValue().toString(), QStringLiteral( "xxx" ) );
```

Comment:
you can setup dynamically a config and retrieve save id as in https://github.com/qgis/QGIS/blob/master/tests/src/core/testqgsauthmanager.cpp#L248